extension/recording-a-log.md

```diff
@@ -14,42 +14,44 @@ Log recording is the core functionality provided by Fiddler Jam. A **Log** in Fi
 
 1. Launch **Fiddler Jam** by clicking on the extension icon ![Extension main icon](../images/ext/ext-icons/small-logo.png).
 
-2. Expand the [**Capture Options**]({%slug fj-capture-options%}) and choose the preferred settings.
+1. (Optional) Expand the [**Advanced Options**]({%slug fj-capture-options%}) and choose the preferred settings. By default, all advanced options are turned on.
```

Comment:
```suggestion
1. (Optional) Expand [**Advanced Options**]({%slug fj-capture-options%}) and choose the preferred settings. By default, all advanced options are enabled.
```

`all advanced options are enabled` - will this continue to be the case in the future? Do we need this sentence?